roles/ceph-restapi/tasks/docker/pre_requisite.yml

```diff
@@ -33,6 +33,13 @@
     state: started
     enabled: yes
 
+- name: install six
```

Comment:
can you use a `with_items` loop to install pip's packages? so you can incorporate that with the docker-py installation